weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -962,10 +966,29 @@ def _select_objs_query(
             {"project_id": project_id, **parameters},
         )
         result: typing.List[SelectableCHObjSchema] = []
-        for row in query_result.result_rows:
+        for row in query_result:
             result.append(
                 SelectableCHObjSchema.model_validate(
-                    dict(zip(query_result.column_names, row))
+                    dict(
+                        zip(
+                            [
```

Comment:
Could be a tuple? With `SELECT *` there is also a chance of a mismatch if we change the schema, I think it would be better to construct the query from this list too.

Reply:
fixed in next commit